test/integration/suite.js

```diff
@@ -22,11 +22,13 @@ module.exports = function(knex) {
     require('./builder/unions')(knex);
     require('./builder/joins')(knex);
     require('./builder/aggregate')(knex);
-    require('./builder/updates')(knex);
-    require('./builder/transaction')(knex);
+    require('./builder/updates')(knex);*/
+    require('./builder/transaction')(
+      knex
+    ); /*
     require('./builder/deletes')(knex);
     require('./builder/additional')(knex);
-    require('./datatype/bigint')(knex);
+    require('./datatype/bigint')(knex);*/
```

Comment:
FYI: You can also run specific test cases by appending `.only` to the declaration.  Ex:

```
it.only('tests whatever', function() {

});
```

Reply:
yes, thanks, but i was too lazy to check is where another describe nesting

Reply:
(this not intended to be committed lol)

Reply:
Yeah -- I figured as much.  lol